pkg/cmd/cli/cmd/request_project.go

```diff
@@ -61,7 +61,7 @@ func NewCmdRequestProject(name, fullName, oscLoginName, oscProjectName string, f
 	}
 	cmd.SetOutput(out)
 
-	cmd.Flags().StringVar(&options.DisplayName, "display-name", "", "project display name")
+	cmd.Flags().StringVar(&options.DisplayName, "displayName", "", "project display name")
```

Comment:
do not change this....

osc new-project foo --display-name="My name" is consistent with how we separate words on all command line actions.